jmix-flowui/flowui/src/main/java/io/jmix/flowui/facet/urlqueryparameters/GenericFilterUrlQueryParametersBinder.java

```diff
@@ -222,12 +228,38 @@ protected String deserializeConfigurationId(String configurationParam) {
     protected List<FilterComponent> deserializeConditions(List<String> conditionParams, DataLoader dataLoader) {
         List<FilterComponent> conditions = new ArrayList<>(conditionParams.size());
         for (String conditionString : conditionParams) {
-            conditions.add(parseCondition(conditionString, dataLoader));
+            FilterComponent filterComponent = parseCondition(conditionString, dataLoader);
+            if (isNotEligible(dataLoader, filterComponent)) {
```

Comment:
It looks like using `continue` to skip a single line of code is redundant. Let's invert `isNotEligible` to `isEligible`/`isPermitted` and add condition if `true`.